NitroxServer/ServerAutoFacRegistrar.cs

```diff
@@ -40,21 +39,18 @@ private static void RegisterCoreDependencies(ContainerBuilder containerBuilder)
             containerBuilder.RegisterType<ConsoleCommandProcessor>().SingleInstance();
 
             containerBuilder.RegisterType<LiteNetLibServer>().SingleInstance();
-            containerBuilder.RegisterType<LidgrenServer>().SingleInstance();
             
             containerBuilder.Register<Communication.NetworkingLayer.NitroxServer>(ctx =>
             {
                 ServerConfig config = ctx.Resolve<ServerConfig>();
```

Comment:
Remove the code in this code block and resolve to LitenetLibServer

```suggestion
       return ctx.Resolve<LiteNetLibServer>();
```

Reply:
Also resolved in the latest commit